plugins/woocommerce/src/Internal/Admin/ShippingLabelBanner.php

```diff
@@ -60,7 +60,9 @@ private function should_show_meta_box() {
 			$incompatible_plugins = class_exists( '\WC_Shipping_Fedex_Init' ) ||
 				class_exists( '\WC_Shipping_UPS_Init' ) ||
 				class_exists( '\WC_Integration_ShippingEasy' ) ||
-				class_exists( '\WC_ShipStation_Integration' );
+				class_exists( '\WC_ShipStation_Integration' ) ||
+				class_exists( '\Automattic\WCShipping\Loader' ) ||
+				class_exists( '\Automattic\WCTax\Loader' );
```

Comment:
Adding both `Automattic\WCShipping\Loader` `Automattic\WCTax\Loader` to `incompatible_plugins` makes sense.